src/kiosk-mode.ts

```diff
@@ -948,6 +959,11 @@ class KioskMode implements KioskModeRunner {
     event.stopImmediatePropagation();
   }
 
+  protected blockContextMenuHandler(event: Event) {
+    event.preventDefault();
+    event.stopImmediatePropagation();
+  }
+
```

Comment:
`blockContextMenuHandler` and `blockToggleMenuGesture` now do the same, it would be good to create a single function (I propose `blockEventHandler`) and use it for both event listeners (and can be reused if more event blockers are needed in the future).